### DISCUSSION TITLE
organizzazione fisica - 10

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#89] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/e761ef4d-1d5e-4d2e-91a6-a229f32c67e9)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/74c7af44-ef15-40fa-98dc-f08b06021ee3)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/83a98365-1d67-4db1-8b87-3c5b6b42b72a)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/45ecae8f-3b20-4d1f-9ee4-68c94a4555d0)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/7e2ad738-0842-410c-9907-720d2ed99cc6)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-07 105945](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/70302a8b-1f41-459b-a9e9-5f8beb7b9e1b)
![Immagine 2024-02-07 105958](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/8acfd5f9-0609-4cc6-ba0d-fd7c7e76ecfa)
![Immagine 2024-02-07 110010](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/b58e77e8-6141-4880-9c43-471850fe40f9)
